tests/llms/test_openai.py

```diff
@@ -1,9 +1,14 @@
+from pathlib import Path
```

Comment:
these tests all seem great. did you mean to include them with this PR?  or should they be a distinct set?

Reply:
👍